extensions/amp-carousel/0.1/scrollable-carousel.js

```diff
@@ -321,7 +321,7 @@ export class AmpScrollableCarousel extends BaseCarousel {
    * @private
    */
   withinWindow_(pos, callback) {
-    const containerWidth = this.element.getLayoutWidth();
```

Comment:
Are we guaranteed that these sync measures won't cause reflows/jank? Shouldn't we still be relying on vysnc?

Reply:
Not in this case, because:

1. There's more measuring code is peppered there, which negates any benefits.
2. It's a component in the deprecation track.